### PR TITLE
Added Material theme as parent, Added request permissions , fixed signout button 

### PR DESCRIPTION
**Pull request by mehtamanan0**
**1.**  Material theme was added as a parent for Theme.amahi adding material progressbars ,dialogs etc   
          as well as some subtle animations in the navigation drawer

**2**  Added request permissions from android sdk 23 for storage 
         screenshot attached below:
screenshot_20170303-035309

**3.** Fixed the signout button mentioned in bug [Bug #1644 ](https://bugs.amahi.org/issues/1644), previously when the signout button was clicked it used to go back to the 
         homescreen now i have added an Intent to go back to the login Screen.

### CODE REVIEW

File: src/main/java/org/amahi/anywhere/activity/NavigationActivity.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ private ActionBarDrawerToggle buildNavigationDrawerToggle() {
 		return new ActionBarDrawerToggle(
 			this,
 			getDrawer(),
-			R.drawable.ic_drawer,
+	        R.drawable.ic_ic_ham_menu,
```
Comment (cpg) [inline src/main/java/org/amahi/anywhere/activity/NavigationActivity.java]: Pleas indent this properly. If you review the code you can see it's off.

**Comment by mehtamanan0** [comment]
@cpg Thank you for reviewing my code i have indented the file properly

File: src/main/java/org/amahi/anywhere/fragment/ServerFilesFragment.java [hunk 2]
```diff
@@ -190,6 +198,32 @@ public boolean onActionItemClicked(ActionMode actionMode, MenuItem menuItem) {
...
+							if (shouldShowRequestPermissionRationale(Manifest.permission.WRITE_EXTERNAL_STORAGE)) {
+										} else {
+											requestPermissions(new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE}, CALLBACK_NUMBER);
+								}
+				} else {
+					startFileSharing(getCheckedFile());
+				}
+		}
+
+				@Override
```
Comment (cpg) [inline src/main/java/org/amahi/anywhere/fragment/ServerFilesFragment.java]: does this line need indentation also? it's in the middle of the code, and I suspect that's not too clean.

I have seen this: `@Override public boolean ....`

Comment (cpg) [inline src/main/java/org/amahi/anywhere/fragment/ServerFilesFragment.java]: I see another one below in the same diff chunk. Please pay attention to detail.

**Comment by mehtamanan0** [comment]
@cpg Sorry for not paying attention i've indented it properly now.

**Comment by cpg** [comment]
attention to details ..
```
.../android-1/src/main/java/org/amahi/anywhere/fragment/ServerFilesFragment.java
Error:(87, 31) error: '{' expected
Error:(89, 10) error: ',', '}', or ';' expected
Error:(89, 24) error: '}' expected
Error:(91, 3) error: not a statement
Error:(91, 7) error: ';' expected
Error:(91, 9) error: not a statement
Error:(91, 26) error: ';' expected
```

**Comment by mehtamanan0** [comment]
@cpg are you still getting some errors?

**Pull request merged**